client/src/views/login/Login.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <section class="login">
+    <h1>Bloom</h1>
+    <h2>Welkom</h2>
+    <form v-if="!submitted" action="/" method="POST" enctype="application/x-www-form-urlencoded">
+      <fieldset>
+        <legend>
+          <label for="emailAddress">
+            <input
+              type="text"
+              placeholder="E-mailadres"
+              id="emailAddress"
+              v-model="user.emailAddress"
+              name="emailAddress"
+            />
+          </label>
+
+          <label for="password">
+            <input
+              type="password"
+              placeholder="Wachtwoord"
+              id="password"
+              v-model="user.password"
+              name="password"
+            />
+          </label>
+          <button @click="login">Inloggen</button>
+          <router-link to="/forgot-password" active-class="forgot-password"
+            >Wachtwoord vergeten</router-link
+          >
+          <router-link to="/register" active-class="register">Ik heb nog geen account</router-link>
+        </legend>
+      </fieldset>
+    </form>
+  </section>
+</template>
+
+<script>
+import DataService from "../../services/DataService.js";
+
+export default {
+  name: "Login",
+  data() {
+    return {
+      user: {
+        // remove?
+        id: null,
+        title: "",
+        description: "",
+        published: false,
+      },
+      submitted: false,
+    };
+  },
+  methods: {
+    login() {
+      var data = {
+        emailAddress: this.user.emailAddress,
+        password: this.user.password,
+      };
+
+      DataService.loginUser(data)
+        .then((response) => {
+          console.log(response);
+          this.tutorial.id = response.data.id;
+          console.log(response.data);
+          this.submitted = true;
+        })
+        .catch((err) => {
+          console.log(err);
+        });
+    },
+
+    newTutorial() {
+      this.submitted = false;
+      this.tutorial = {};
```

Comment:
same for this

Reply:
Where is this function for? Can we optimise this?

Reply:
I am not sure for what this function is since I did not write it :), I am assuming this is a functin to create a new user?

Reply:
Ah okay, I think it's a sort of state function. We will check it in a meeting soon ;)